app/commands/get_user_review_request_metrics.rb

```diff
@@ -0,0 +1,35 @@
+class GetUserReviewRequestMetrics < PowerTypes::Command.new(:github_user, :limit_month)
+  def perform
+    entry_count = 0
+    total_time = 0
+    review_requests_dic = Hash.new
+    limit_date = (Date.today - @limit_month.to_i.months).beginning_of_day
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Rails/Date: Do not use Date.today without zone. Use Time.zone.today instead.